Integrations/FortiGate/FortiGate.py

```diff
@@ -94,6 +94,44 @@ def create_addr_string(list_of_addr_data_dicts):
     return addr_string
 
 
+def convert_arg_to_int(arg_str, arg_name_str):
+    try:
+        arg_int = int(arg_str)
+    except Exception:
+        return_error("Error: {0} must have an integer value.".format(arg_name_str))
+    return arg_int
+
+
+def prettify_date(date_string):
+    date_string = date_string[:-5]  # remove the .000z at the end
+    date_parts = date_string.split("T")
```

Comment:
why split and format when you can just replace?
```suggestion
    date_parts = date_string.replace('T', ' ')
```